src/_pytest/logging.py

```diff
@@ -72,24 +72,80 @@ class PercentStyleMultiline(logging.PercentStyle):
     formats the message as if each line were logged separately.
     """
 
+    def __init__(self, fmt, auto_indent):
+        super().__init__(fmt)
+        self._auto_indent = auto_indent
+
     @staticmethod
     def _update_message(record_dict, message):
         tmp = record_dict.copy()
         tmp["message"] = message
         return tmp
 
+    @staticmethod
+    def _get_auto_indent(record, default_auto_indent) -> int:
+        """Determines the current auto indentation setting
+
+        Specify auto indent behavior (on/off/fixed) by passing in
+        extra={"auto_indent": [value]} to the call to log.log() or
+        using a --log-auto-indent [value] command line or config option.
+        Default behavior is auto-indent on.
+
+        Using the string "True" or "on" or the boolean True as the value
+        turns auto indent on, using the string "False" or "off" or the
+        boolean False or the int 0 turns it off, and specifying a
+        positive integer fixes the indentation position to the value
+        specified.
+
+        Any other values for the option are invalid, and will silently be
+        converted to "on".
+        """
+        out_auto_indent = -1
+        if hasattr(record, "auto_indent"):
+            # passed in from the "extra={}" kwarg on the call to log.log()
+            in_auto_indent = record.auto_indent
+        elif default_auto_indent is not None:
+            in_auto_indent = default_auto_indent
+        else:
+            in_auto_indent = -1
```

Comment:
This branch is only reachable if the user didn't specify a parameter to log-cli-level. Maybe we don't need distinguish between default_auto_indent being None and not-None.